bifrost/export/pynn.py

```diff
@@ -38,4 +36,18 @@ class PyNNNeuronTypes(Enum):
 def export_structure(layer: Layer) -> Statement:
     ratio = float(layer.shape[1]) / layer.shape[0]
     return Statement(f"Grid2D({ratio})",
-                     imports=['from pyNN.space import Grid2D'])
\ No newline at end of file
+                     imports=['from pyNN.space import Grid2D'])
+
+
+def export_constraints(constraints: Dict[str, Any]) -> Statement:
+    if 'max_neurons' in constraints:
+        return export_max_neurons_per_core(constraints['max_neurons'])
+
```

Comment:
Would it make sense to raise an exception or log a warning if we don't support the constraint in the dict?

Reply:
Yes to both, I don't know what I was thinking in splitting configs and constraints 😓  I'll make the changes